src/Agoda.IoC.NetCore/StartupExtension.cs

```diff
@@ -34,6 +42,21 @@ public static IServiceCollection AutoWireAssembly<T>(this IServiceCollection ser
                     attr => new RegistrationContext(attr, x.ToType, isMockMode)
                 ))
                 .ToList();
+
+            if (registrations.Any(x => !x.Validation.IsValid))
+            {
+                var exceptions  = new ContainerRegistrationContextException("There are validations errors!!!");
```

Comment:
This is a single object, why does the name need to be plural? To me that implies a collection.

Reply:
will fix this thanks